### PR TITLE
enable build of pypi package

### PR DESCRIPTION
**Pull request by schiele**

### CODE REVIEW

File: ldraw_to_scad
Comment (dannystaple) [inline ldraw_to_scad:10]: It might be nice to name these things camel_case, both the function and it's parameters.

File: ldraw_to_scad
Comment (dannystaple) [inline ldraw_to_scad:21]: This prints skipping, but still sets `lst[key]` - doesn't quite sound right to me.

File: ldraw_to_scad
Comment (schiele) [reply ldraw_to_scad:10]: Not really new code but just moved around. But yes, I could change that as suggested.

File: ldraw_to_scad
Comment (schiele) [reply ldraw_to_scad:21]: Note that this is the case where two files with identical file names but different extensions (like foo.mpd and foo.ldr) exist causing a collision in the target file, both resolving to foo.scad. Therefore we skip the one with lower priority (foo.ldr in the example) and store the extension of the _new_ file with higher priority. I acknowledge though that this likely deserves some extra comment in the code. ;)

File: setup.cfg
Comment (schiele) [inline setup.cfg:5]: Danny, do you want to be listed as co-maintainer of the pypi packages as well or prefer to stay out here? If you want to get listed you should send me your pypi account name.

Comment (dannystaple) [reply setup.cfg:5]: Hello, it would be nice to be listed too. The account name is dannystaple (now I've created one).

Comment (schiele) [reply setup.cfg:5]: Okay, this gets on the list of pending changes for this draft. For now, you can see the current test version on https://test.pypi.org/project/ldraw-to-scad/. Note that this version is not working yet since I did not package the OpenSCAD library code yet, therefore a pip-installed package will fail to find that data with this test version unless when called from a checked-out source repository, which for sure a regular user would not do.

Comment (dannystaple) [reply setup.cfg:5]: Nice, looking forward to seeing this project there.

File: ldraw_to_scad
Comment (schiele) [reply ldraw_to_scad]: Comment added.

File: setup.cfg
Comment (schiele) [reply setup.cfg]: Closer inspection revealed that setuptools allow only exactly one author and one maintainer. As such, I added you as the author and myself as the maintainer, which most likely is the closest match to reality.
I think this is ready now for final inspection before merging. Maybe you also want to try the package on the test server from the link above on a system that does not have access to the sources (since this is how a regular user would see it). This should install in a clean way and operate correctly. After merging this I would make a first official release to the regular pypi repository instead of the test one.

**Review by dannystaple — APPROVED** [review]

**Pull request merged**
